apache/superset
# feat: New Limit Selector

### SUMMARY

Added a new limit selector for SQL Toolbar revamce

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

<img width="128" alt="Screen Shot 2020-12-15 at 2 31 54 PM" src="https://user-images.githubusercontent.com/27827808/102263834-d6128100-3ee2-11eb-8471-6e3dc6ad8035.png">

### TEST PLAN

1. Goto sqllab
2. Hover limit dropdown
3. Select a limit
4. Run query
5. Verify query return same number of rows

### ADDITIONAL INFORMATION
